troubleshooting.md

```diff
@@ -0,0 +1,570 @@
+# Troubleshooting Guide for BM IPI Deployments
+
+* [Bootstrap VM issues](#bootstrap-vm-related-issues)
+
+* [Following the Installation](#following-the-installation) 
+
+* [Node Related Issues](#node-related-issues)
+
+* [NTP out of sync](#ntp-out-of-sync)
+
+* [Registry Issues](#registry-issues) 
+
+* [IPv6 Disconnected Install Troubleshooting](#ipv6-disconnected-install-troubleshooting)
+
+* [Misc Issues](#misc-issues) 
+
+**Pre-requisites:** 
+
+First and foremost the BM installations follows a general architecture:
+* provisioner node 
+* master node 
+* worker node
+
+Before running the installation, please make sure you have a cleaned the previous files/manifests.
+
+# Bootstrap VM related issues 
+
+* Cannot login to the Bootstrap VM
+```
+$ ssh core@172.22.0.2
+Permission denied (publickey,password,keyboard-interactive).
+```
+
+Generally if the SSH keys are not updated in `install-config.yaml` file you cannot login to the bootstrap VM, add/update the correct SSH keys. 
+
+Make sure this file has the SSH keys of the provisioner node, if not you can't login into any of the nodes after deployment is completed.
+
+After the installation is triggered, Check/verify bootstrap VM is up and running(may take few minutes) from Provisioner node: 
+
+```
+[kni@provisioner]$ sudo virsh list
+ Id    Name                           State
+----------------------------------------------------
+ 12    kni7-xf6fq-bootstrap           running
+```
+
+If there is no sign of Bootstrap VM running on the provioner node, check if the libvirtd is running(if not, start the service) on the provisioner node. 
+
+```
+systemctl status libvirtd
+● libvirtd.service - Virtualization daemon
+   Loaded: loaded (/usr/lib/systemd/system/libvirtd.service; enabled; vendor preset: enabled)
+   Active: active (running) since Mon 2020-02-03 13:08:51 EST; 2 weeks 0 days ago
+     Docs: man:libvirtd(8)
+```
```

Comment:
I get what your trying to say here but its odd because why would I even check this? Only reason I'd check this is if my install failed. Then I'd go attempt to see why. If my install was a success I would never even bother checking whether my bootstrap VM launched or not. And saying check after install is triggered is inaccurate because it doesn't happen immediately.

Reply:
So, the only reason is not every time the bootstrap VM comes up, I mentioned that after few minutes after installation is triggered if this VM is not UP, it's better to check the status of libvirtd(which might be in an inactive state).